code/modules/mob/living/simple_animal/hostile/terror_spiders/chem.dm

```diff
@@ -0,0 +1,100 @@
+#define SOLID 1
+#define LIQUID 2
+#define GAS 3
+#define FOOD_METABOLISM 0.4
+#define REM REAGENTS_EFFECT_MULTIPLIER
+
+
+// Terror Spider, Black, Deadly Venom
+
+/datum/reagent/terror_black_toxin
+	name = "Black Widow venom"
+	id = "terror_black_toxin"
+	description = "An incredibly toxic venom injected by the Black Widow spider."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.1
+
+/datum/reagent/terror_black_toxin/on_mob_life(var/mob/living/M as mob)
+	if(!M)
+		M = holder.my_atom
+	if(volume >= 45)
+		// bitten 4 or more times, whole body goes into shock/death
+		// total damage: 12, human health 150 until crit, = 12.5 ticks, = 25s until death
```

Comment:
all of these comments are really unnecessary, we can expect people to know how effective `adjustToxLoss` is by looking at mob code themselves